linera-examples/fungible2/src/contract.rs

```diff
@@ -25,14 +25,14 @@ linera_sdk::contract!(FungibleToken<ViewStorageContext>);
 impl Contract for FungibleToken<ViewStorageContext> {
     type Error = Error;
     type Storage = ViewStateStorage<Self>;
+    type InitializationArguments = BTreeMap<AccountOwner, Amount>;
 
     async fn initialize(
         &mut self,
         context: &OperationContext,
-        argument: &[u8],
+        argument: Self::InitializationArguments,
     ) -> Result<ExecutionResult, Self::Error> {
-        let mut accounts: BTreeMap<AccountOwner, Amount> =
-            bcs::from_bytes(argument).map_err(Error::InvalidInitialState)?;
+        let mut accounts = argument;
```

Comment:
Of course this will be simplified.

Reply:
This is just essentially aliasing the argument.

Reply:
I know but in this case one can use `mut argument` in the function header